### DISCUSSION TITLE
Web Development technologies

### DISCUSSION THREAD

**Discussion by DuajDiaz** [DuajDiaz/Travel-Website#66] [Q&A] [answered]
What are some of the Web Development technologies that you know?

**Comment by Bunny77K9** [comment, marked as answer]
This question is very common in a Web Development interview. The answer is based on your learning, experience, and proficiency in this field. However, a good [Web Developer](https://intellipaat.com/blog/what-is-web-developer/) will have profound working skills in any of the following technologies:

    HTML
    CSS
    JavaScript
    SQL
    Python
    PHP

**Comment by Pavitra554** [comment]
Currently
Html
Css
Js
Ts
TailwindCss
React.js
Next.js
Firebase
Framer motion
Redux
..
